src/e2e/__tests__/caret.ts

```diff
@@ -54,9 +58,12 @@ describe('caret testing', () => {
 
     await page.keyboard.press('Enter')
     await paste(page, [''], importText)
-    await setCursor(page, ['Don\'t stay awake for too long', 'I don\'t wanna fall asleep'], { offset: 10 })
-    await clickBullet(page, 'Don\'t stay awake for too long')
 
+    await waitForEditable(page, 'I don\'t wanna fall asleep')
+    const editableNodeHandle = await getEditable(page, 'I don\'t wanna fall asleep')
+    await clickWithOffset(page, editableNodeHandle, { horizontalClickLine: 'left', offset: 10 })
+
+    await clickBullet(page, 'Don\'t stay awake for too long')
```

Comment:
This instance of `clickBullet` is still valid since the test is actually about clicking on a bullet.